fflas-ffpack/ffpack/ffpack.h

```diff
@@ -599,8 +599,21 @@ namespace FFPACK {
     PLUQ (const Field& F, const FFLAS::FFLAS_DIAG Diag,
           const size_t M, const size_t N,
           typename Field::Element_ptr A, const size_t lda,
-          size_t*P, size_t *Q, size_t BCThreshold = __FFLASFFPACK_PLUQ_THRESHOLD);
```

Comment:
why did you get rid of the threshold parameter?
You should keep it and implement PLUQ in the same way as fsytrf is done with threshold and parseqhelper or without parseqhelper